command/src/channel.rs

```diff
@@ -569,14 +560,14 @@ mod tests {
         println!("Waiting 200 milliseconds…");
         thread::sleep(std::time::Duration::from_millis(200));
 
-        writing_channel.write_message(&Serializable(200));
+        writing_channel.write_message(&Serializable(200)).unwrap();
```

Comment:
To me, this should be replaced by an `expect`

Reply:
done